src/index.scss

```diff
@@ -1 +1,5 @@
+@font-face {
```

Comment:
Probably, we need to create an file inside /assets/typography.scss and import it inside index.ts